superbit_lensing/shear_profiles/shear_plots.py

```diff
@@ -2,22 +2,45 @@
 from matplotlib import rc,rcParams
 import matplotlib.pyplot as plt
 from astropy.table import Table
-import pdb
+import pudb
+
+def compute_alpha(nfw, radius, gtan, variance):
```

Comment:
I ended up adding a `compute_shear_bias()` function to `annular_jmac.py` in the `resample_redshift` branch I've been working in. The shear bias and its uncertainty are computed therein, and adds those values to the metadata of the file f/k/a `run_name_annular_shear_tab.fits` (now `run_name_shear_profile.fits`?). 

For now, we can merge this into `master` as you've written it, but when we integrate the `redshift_resample` branch, the shear bias and uncertainty values can just be accessed from the table.

Reply:
Just realized that `compute_shear_bias()` isn't something I can import yet as it isn't in `master`, so I'll keep it for now as you suggested.